metadata.json

```diff
@@ -1,13 +1,13 @@
 {
```

Comment:
It looks like this Metadata is incorrect. The version number has gone backwards and the source link has been removed.

It might be worth considering if this is a major version bump as well, I'm not sure from looking that it's 100% backwards compatible?